assets/src/edit-story/components/fontPicker/test/fontPicker.js

```diff
@@ -24,16 +24,14 @@ import { act, fireEvent, waitFor } from '@testing-library/react';
  */
 import FontPicker from '../';
 import { FontProvider } from '../../../app/font';
-import APIContext from '../../../app/api/context';
 import { renderWithTheme } from '../../../testUtils';
 import { curatedFontNames } from '../../../app/font/curatedFonts';
 import fontsListResponse from './fontsResponse';
 
-async function getFontPicker(options) {
-  const getAllFontsPromise = Promise.resolve(fontsListResponse);
-  const apiContextValue = {
-    actions: {
-      getAllFonts: () => getAllFontsPromise,
+function getFontPicker(options) {
```

Comment:
These tests are still failing. I am going to leave this for the weekend and come fresh at it on Monday. If you have a bright idea @swissspidy, let me know.

Reply:
You're using the `FontProvider` directly, but still trying to provide your own context state value. I've updated it to use the `FontContext` instead and completely bypassing the `FontProvider` in e74adf99e.